packages/errors/src/test-utils.ts

```diff
@@ -1 +0,0 @@
-export * from './test-utils/expect-to-throw-fuel-error';
```

Comment:
This export should remain in this file, following [this](https://github.com/FuelLabs/fuels-ts/pull/1230/files/2a16499381e169379c3625cf409085911b1ec298..b29792c3b4d5106f7335110d0ebfaabc1cdee878#r1322872516).